apps/website/src/components/Placeholder.astro

```diff
@@ -0,0 +1,39 @@
+---
+import { SvgInfoCircular } from '@itwin/itwinui-icons-react';
+
+export type Props = {
+  /** ending of the url for a specific storybook page */
+  componentPageName: string;
+};
+
+const { componentPageName } = Astro.props;
+---
+
+<div class='alert' data-iui-status='informational'>
+  <SvgInfoCircular className='iui-alert-icon' aria-hidden='true' />
+  <span class='iui-alert-message'
+    >This page has not yet been finished and is being worked on. In the meantime, visit <a
+      class='iui-alert-link'
+      href=`https://itwin.github.io/iTwinUI/react/?path=/story/${componentPageName}`>storybook</a
+    > to see more examples.
+  </span>
+</div>
+
+<style lang='scss'>
+  .alert {
```

Comment:
styles are automatically scoped to the component, so you can just use `div` here instead of `.alert`